src/components/Accordion.js

```diff
@@ -32,21 +64,27 @@ const selectElementByClassName = (element, cssName) => {
   }
 };
 
-const menuAction = element => {
-  var mainDiv = element.closest(".dg_accordion");
+const menuAction = accordionHeaderText => {
```

Comment:
Not blocking, the previous variable name actually makes more sense to me than this one. To me it seems like this is a string "My Awesome Accordion Header Text" instead of the actual element.

I think this is probably a good thing to discuss and document as a pattern.

I propose in this case


When a variable is set to an [Element] (https://developer.mozilla.org/en-US/docs/Web/API/Element) the pattern would be as follows. (Use `Elm` as a suffix).

```js
const accordionHeaderElm = document.getElementById('accordion-header-1');
```

When a variable is set to a [NodeList](https://developer.mozilla.org/en-US/docs/Web/API/NodeList) the pattern would be as follows. (Use `Elms` as a suffix).

```js
const accordionHeaderElements = document.querySelectorAll('.accordion-header');
```


@tmccoy529 @sgrg1 @theilman7 what do you think?

Once there is agreement, we will move the decision to this document, https://github.com/baltimorecounty/dotgov-components/wiki/Naming-Things where will maintain our naming things standards, until we find a better home for them.

Reply:
@tmccoy529 just brought up a good point on why we have the distinction between an array and  NodeList:

The difference between an array and a NodeList which docuemnt.querySelectorAll returns is that the NodeList doesn't contain all of the array methods that an array method does.

So for instance, you cannot `map` a nodelist, but you can an array

```js
var dogs = ['test', 'test1'];

dog.map(myMapperFunction); // works

var dogs = document.querySelectorAll('.dogs');

dogs.map // does not work

```

Reply:
@martypowell @theilman7 @tmccoy529 understood Elm vs Elms

Reply:
This seems to make sense to me as it will make things stand out more as elements and therefore could be easier to troubleshoot if something needs to be addressed at a later date.

Reply:
per our convo, this has been documented here, https://github.com/baltimorecounty/dotgov-components/wiki/Naming-Things

please provide any feedback @baltimorecounty/front-end-developers .